### DISCUSSION TITLE
App doesn't launch

### DISCUSSION THREAD

**Discussion by krzyywyy** [CarlGao4/Demucs-Gui#87] [Q&A]
I have little problem. The application doesn't start, there are no log files, no errors. What could i do?

**Comment by DilsonAbraham** [comment]
I have the same issue, but when I checked the log folder this is what I see.

Can the developer help me out on this?

_

> 2024-10-01 22:07:13,458 (GuiMain.py) (Line 1763) [INFO] : Python version: 3.11.6 | packaged by conda-forge | (main, Oct  3 2023, 10:29:11) [MSC v.1935 64 bit (AMD64)]
> 2024-10-01 22:07:13,459 (GuiMain.py) (Line 1764) [INFO] : Demucs GUI version: 1.1
> 2024-10-01 22:07:13,481 (GuiMain.py) (Line 1765) [INFO] : System: Windows-10-10.0.26100-SP0
> 2024-10-01 22:07:13,482 (GuiMain.py) (Line 1766) [INFO] : Architecture: 64bit
> 2024-10-01 22:07:13,482 (GuiMain.py) (Line 1767) [INFO] : CPU: AMD64 Family 25 Model 33 Stepping 0, AuthenticAMD
> 2024-10-01 22:07:13,482 (GuiMain.py) (Line 1768) [INFO] : CPU count: 32
> 2024-10-01 22:07:13,489 (GuiMain.py) (Line 1769) [INFO] : System memory: 34272837632 (31.919GB)
> 2024-10-01 22:07:13,494 (GuiMain.py) (Line 1772) [INFO] : System free memory: 24547491840 (22.862GB)
> 2024-10-01 22:07:13,773 (GuiMain.py) (Line 1775) [INFO] : System swap memory: 2147483648 (2GB)
> Traceback (most recent call last):
>   File "GuiMain.py", line 1778, in <module>
>   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
>   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
>   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
>   File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
>   File "find_device_win.py", line 103, in <module>
>   File "shared.py", line 271, in Popen
>   File "subprocess.py", line 1026, in __init__
>   File "subprocess.py", line 1538, in _execute_child
> FileNotFoundError: [WinError 2] The system cannot find the file specified

_

**Reply by CarlGao4** [reply]
The wmic command used to detect hardware information has been removed in latest Windows 11. I'll fix this in the next version. Meanwhile, you can open System Settins (`WIN` + `I`), go to System -> Optional features, and search for wmic and install it.

**Reply by DilsonAbraham** [reply]
That helped. Appreciate your support.
Will continue to support you in whatever way possible to make this is an even greater project. 
Please keep up the good work.
